# CMake error at CMakeLists.txt:117(message): The C++ compiler does not support C++11(e.g.std:unique_ptr)
# https://discourse.cmake.org/t/cmake-error-at-cmakelists-txt-117-message-the-c-compiler-does-not-support-c-11-e-g-std-unique-ptr/3774

diff --git a/CMakeLists.txt b/CMakeLists.txt
index dfbb38db..229d50a2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -82,17 +82,17 @@ if(NOT DEFINED CMAKE_CXX_STANDARD AND NOT CMake_NO_CXX_STANDARD)
     endif()
   endif()
 endif()
-if(NOT CMake_TEST_EXTERNAL_CMAKE)
-  # include special compile flags for some compilers
-  include(CompileFlags.cmake)
-
-  # check for available C++ features
-  include(${CMake_SOURCE_DIR}/Source/Checks/cm_cxx_features.cmake)
-
-  if(NOT CMake_HAVE_CXX_UNIQUE_PTR)
-    message(FATAL_ERROR "The C++ compiler does not support C++11 (e.g. std::unique_ptr).")
-  endif()
-endif()
+# if(NOT CMake_TEST_EXTERNAL_CMAKE)
+#   # include special compile flags for some compilers
+#   include(CompileFlags.cmake)
+#
+#   # check for available C++ features
+#   include(${CMake_SOURCE_DIR}/Source/Checks/cm_cxx_features.cmake)
+#
+#   if(NOT CMake_HAVE_CXX_UNIQUE_PTR)
+#     message(FATAL_ERROR "The C++ compiler does not support C++11 (e.g. std::unique_ptr).")
+#   endif()
+# endif()
 
 # Inform STL library header wrappers whether to use system versions.
 configure_file(Utilities/std/cmSTL.hxx.in Utilities/cmSTL.hxx @ONLY)
